### DISCUSSION TITLE
领普人在传感器建议把红外和雷达分开作为两个传感器

### DISCUSSION THREAD

**Discussion by Skyearn** [XiaoMi/ha_xiaomi_home#296] [Ideas]
融合以后只有一个有人无人则无法作为自动化的触发器，比如人长期在区域内活动，就没有从无人 -> 有人的变化。而众所周知触发器依赖的是状态变化作为条件。以开灯为例：如果我需要在区域内有人且亮度低于某个阈值时开灯。那么我们可以用有人无人、光照度低于某个阈值作为触发器。但是红外和雷达融合后这两个都只会触发一次。并不是一个合格的触发器。还需要额外添加一个红外传感器来作为触发器。但是人在本来就既有红外又有雷达。
拆开来，使用红外触发，再判断雷达，比融合在一起一直有人更符合实际情况。
![126B0F8C-2DB6-4B99-A55F-E2DB6A49B906_1_201_a](https://github.com/user-attachments/assets/47cb891a-4152-48c6-a1b3-6e15af206171)

**Comment by Skyearn** [comment]
#243

**Comment by NOisi-x** [comment]
这个设备在[MIoT-Spec-V2]中没有分开红外和雷达，所以米家无法实现这个需求，你得找领普才行
